source/backend/api/Pims.Api.csproj

```diff
@@ -46,6 +46,7 @@
     <PackageReference Include="Microsoft.Extensions.PlatformAbstractions" Version="1.1.0" />
     <PackageReference Include="Microsoft.IO.RecyclableMemoryStream" Version="2.3.2" />
     <PackageReference Include="NetTopologySuite.IO.GeoJSON4STJ" Version="4.0.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
Do we really need this now? I though we were using the default **Microsoft.Text.Json** serializer...

Reply:
maybe not, it was already installed I've just updated it because it was giving a vulnerability warning.